### DISCUSSION TITLE
Response length

### DISCUSSION THREAD

**Discussion by deedeeharris** [lspahija/AIUI#25] [Q&A] [answered]
Your app is awesome. Where can I set the response limit? I noticed that it's very short.

Thanks

**Comment by lspahija** [comment, marked as answer]
Thank you!

This is the line of code that affects response length: https://github.com/lspahija/AIUI/blob/main/app/ai.py#L11

It's not currently configurable, but feel free to open a PR that allows users to set the initial prompt through an environment variable.
